components/DropdownAux.tsx

```diff
@@ -0,0 +1,23 @@
+const DropdownAux = ({ submenus, dropdown, mobile }: { submenus: any, dropdown: boolean, mobile?: boolean }) => {

```

Comment:
it's possible to create a specific type for submenus, it's not necessary to use "any".

dropdown?, I understand what dropdown does looking the HeaderItems.tsx and like i said in the HeaderItems component, dropdown is not a good variable name for the dropdown state.